contrib/emacs/hurl-mode.el

```diff
@@ -0,0 +1,136 @@
+;;; hurl-mode.el --- Major mode for hurl  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022 Azzam S.A
```

Comment:
@jcamiel this need to be changed. For reference:
- https://github.com/spotify/dockerfile-mode/blob/52c6c00da1d31c0b6c29c74335b3af63ed6bf06c/dockerfile-mode.el#L3